web_external/stylesheets/body/imagesPage.styl

```diff
@@ -1,20 +1,110 @@
-/* patch backbone's awkward padding enforcement */
-#g-app-body-container {
-  padding: 0px !important;
-}
-
-#galleryContent {
-  position: relative;
-  width: 100%;
-  min-height: 12em;
-  height: calc(100vh - 12em);
-}
-
-#galleryContent #isic-images-pageContent {
-  position: absolute;
-  top: 0px;
-  left: 0px;
-  height: 100%;
-  width: 100%;
-  overflow: auto;
-}
+#galleryContent
+  position relative
+  min-height 12em
+  height calc(100vh - 13em)
+  display flex
+  width 100%
+  flex-wrap nowrap
+  -ms-user-select none
+  -moz-user-select none
+  -webkit-user-select none
+
+  .flexRow
+    display flex
+    width 100%
+    flex-wrap nowrap
+
+  .flexColumn
+    display flex
+    flex-direction column
+    height 100%
+    flex-wrap nowrap
+
+  .fixedWidth
+    min-width 32em
+    max-width 32em
+    flex-basis 32em
+    overflow-y auto
+
+  .spacer
+  .growing
+    flex-grow 1
+
+  .pane
+    position relative;
+    padding 0.5em;
+    margin 0.5em;
+    border 1px solid #DDDDDD;
+    border-radius 3px;
+
+  /* Study pane */
+  #isic-images-studyPane
+
+  /* Study text pane */
+  #isic-images-studyTextPane
+
+  /* Image wall */
+  #isic-images-imageWall
+    overflow-y auto
+    overflow-x hidden
+    min-width 32em
+
+    image
+      cursor pointer
+
+    rect#highlightOutline
+      fill none
+      stroke #00ABFF
+      stroke-width 4px
+
+  /* Paging pane */
+  #isic-images-pagingPane
+    flex-basis 3em
+    height 3em
+    max-height 3em
+    min-height 3em
+    display flex
+    flex-wrap nowrap
+    align-items center
+
+    .button
+      margin 0.5em
+      cursor pointer
+      border 1px solid transparent
+      border-radius 3px
+      padding 3px
+
+      img
+        width 1em
+        height 1em
+
+        .selected
+          -webkit-filter url(#recolorImageTo00ABFF)
```

Comment:
This is exactly what that weird shim does—it makes SVG filters available in CSS.